package.json

```diff
@@ -24,24 +24,25 @@
     "@types/express": "^4.11.1",
     "@types/jest": "^22.2.2",
     "@types/node": "^9.6.6",
-    "@types/node-fetch": "^2.1.0",
+    "@types/node-fetch": "^2.1.1",
     "@types/validator": "^9.4.0",
+    "abort-controller": "^1.0.2",
     "danger": "^3.1.7",
     "danger-plugin-digitalcitizenship": "^0.3.1",
-    "jest": "^22.4.3",
+    "jest": "^22.4.4",
     "jest-mock-express": "^0.1.1",
     "mock-http-server": "^1.0.0",
     "node-fetch": "https://github.com/bitinn/node-fetch#abort",
     "prettier": "^1.12.1",
     "rimraf": "^2.6.2",
-    "ts-jest": "^22.4.2",
+    "ts-jest": "^22.4.6",
     "tslint": "^5.1.0",
     "tslint-config-prettier": "^1.6.0",
     "tslint-immutable": "^4.5.4",
     "tslint-plugin-prettier": "^1.3.0",
     "tslint-sonarts": "^1.6.0",
     "typedoc": "^0.11.1",
-    "typescript": "^2.8.0",
+    "typescript": "~2.8.4",
```

Comment:
why?

Reply:
otherwise yarn will update to 2.9 which is not compatible with the last version of io-ts

Reply:
yarn doesn't update if you don't update yarn.lock :)

Reply:
I cant because I cannot install node-fetch package (see above) and yarn breaks there (that's why we cannot merge this PR : )